test/powershell/Modules/Microsoft.PowerShell.Management/Get-ComputerInfo.Tests.ps1

```diff
@@ -0,0 +1,1367 @@
+
+    $propertyNames += @("OsArchitecture",
+        "OsBootDevice",
+        "OsBuildNumber",
+        "OsBuildType",
+        "OsCodeSet",
+        "OsCountryCode",
+        "OsCSDVersion",
+        "OsCurrentTimeZone",
+        "OsDataExecutionPrevention32BitApplications",
+        "OsDataExecutionPreventionAvailable",
+        "OsDataExecutionPreventionDrivers",
+        "OsDataExecutionPreventionSupportPolicy",
+        "OsDebug",
+        "OsDistributed",
+        "OsEncryptionLevel",
+        "OsForegroundApplicationBoost",
+        "OsHardwareAbstractionLayer",
+        "OsHotFixes",
+        "OsInstallDate",
+        "OsLanguage",
+        "OsLastBootUpTime",
+        "OsLocale",
+        "OsLocaleID",
+        "OsManufacturer",
+        "OsMaxProcessMemorySize",
+        "OsMuiLanguages",
+        "OsName",
+        "OsNumberOfLicensedUsers",
+        "OsNumberOfUsers",
+        "OsOperatingSystemSKU",
+        "OsOrganization",
+        "OsOtherTypeDescription",
+        "OsPAEEnabled",
+        "OsPagingFiles",
+        "OsPortableOperatingSystem",
+        "OsPrimary",
+        "OsProductSuites",
+        "OsProductType",
+        "OsRegisteredUser",
+        "OsSerialNumber",
+        "OsServerLevel",
+        "OsServicePackMajorVersion",
+        "OsServicePackMinorVersion",
+        "OsSizeStoredInPagingFiles",
+        "OsStatus",
+        "OsSuites",
+        "OsSystemDevice",
+        "OsSystemDirectory",
+        "OsSystemDrive",
+        "OsTotalSwapSpaceSize",
+        "OsTotalVirtualMemorySize",
+        "OsTotalVisibleMemorySize",
+        "OsType",
+        "OsVersion",
+        "OsWindowsDirectory")
+
+    $propertyNames += @("KeyboardLayout",
+        "LogonServer",
+        "PowerPlatformRole",
+        "TimeZone")
+
+    $WindowsPropertyArray = @("WindowsBuildLabEx",
+        "WindowsCurrentVersion",
+        "WindowsEditionId",
+        "WindowsInstallationType",
+        "WindowsProductId",
+        "WindowsProductName",
+        "WindowsRegisteredOrganization",
+        "WindowsRegisteredOwner",
+        "WindowsSystemRoot",
+        "WindowsVersion")
+   
+    if ([System.Management.Automation.Platform]::IsIoT)
+    {
+        Write-Verbose -Verbose -Message "WindowsInstallDateFromRegistry is not supported on IoT."
+    }
+    else
+    {
+        $WindowsPropertyArray += "WindowsInstallDateFromRegistry"
+    }
+
+    $propertyNames += $WindowsPropertyArray
+    
+    return $propertyNames
+}
+
+function New-ExpectedComputerInfo
+{
+    param([string[]]$propertyNames)
+
+    # P-INVOKE TYPE DEF START ******************************************
+    function Get-FirmwareType
+    {
+$signature = @"
+[DllImport("kernel32.dll")]
+public static extern bool GetFirmwareType(ref uint firmwareType);
+"@
+        Add-Type -MemberDefinition $signature -Name "Win32BiosFirmwareType" -Namespace Win32Functions -PassThru
+    }
+
+    function Get-PhysicallyInstalledSystemMemory
+    {
```

Comment:
Everything is artificially indented after this point. Remove the indent if not nested.